README.md

```diff
@@ -30,76 +41,90 @@ $ helm install ohmyglb ohmyglb/ohmyglb
 See [values.yaml](https://github.com/AbsaOSS/ohmyglb/blob/master/chart/ohmyglb/values.yaml)
 for customization options.
 
-### Local Playground
-
-####  Deploy local cluster
-
-```sh
-$ make deploy-local-cluster
+### Local Playground Install
+
+#### Environment prerequisites
+
+ - install **GO 1.14**
+ 
+ - install **GIT**     
+    - ensure SSH key is set for your account; see https://github.com/settings/keys  
+    
+ - install **gnu-sed** if you don't have it  
+    - update $PATH in `~/.bash_profile` by sed path `export PATH=/usr/local/opt/gnu-sed/libexec/gnubin:$PATH` 
+   
+ - install **Docker**
+    - ensure you are able to push/pull from your docker registry
+    - to run multiple clusters reserve 8GB of memory  
+    
+      ![](docs/images/docker_settings.png)
+      <div>
+        <sup>above screenshot is for <strong>Docker for Mac</strong> and that options for other Docker distributions may vary</sup>
+      </div>
+
+ - install **Kubectl** to operate clusters
+     
+ - [install **Helm3**](https://helm.sh/docs/intro/install/) to get charts
+    - add default repo 
+    ```shell script
+    helm repo add stable https://kubernetes-charts.storage.googleapis.com
+    ```
+ 
+ - install **kind** as tool for running local Kubernetes clusters
+    - follow https://kind.sigs.k8s.io/docs/user/quick-start/
+ 
+
+
+#### Running project locally
+
+To spin-up a local environment using two Kind clusters and deploy a test application to both clusters, execute the command below: 
+```shell script
+make deploy-full-local-setup 
 ```
-Creates local [kind](https://github.com/kubernetes-sigs/kind) cluster
-with several workers for more realistic setup.
 
-#### Deploy local ingress
-
-```sh
-$ make deploy-local-ingress
-```
-Creates local nginx ingress with deployment config similar to Rancher.
-IP addresses of workers will be exposed as Ingress addresses.
-It will create proper environment for Ohmyglb testing
 
-#### Deploy gslb operator
+#### Verify installation
+If local setup runs well, check if clusters are correctly installed
 
-```sh
-$ make deploy-gslb-operator
+```shell script
+kubectl cluster-info --context kind-test-gslb1 && kubectl cluster-info --context kind-test-gslb2
 ```
-Operator is packaged as a helm chart at [chart/ohmyglb](/chart/ohmyglb) and its
-configuration is controlled by [chart/ohmyglb/values.yaml](/chart/ohmyglb/values.yaml)
-
-This step will deploy the operator and its dependencies:
-
-* `EtcdCluster` controlled by [etcd-operator](https://github.com/helm/charts/tree/master/stable/etcd-operator)
```

Comment:
I agree with cleaning up readme. But we need to create acrhitecture doc depicting current implementation pieces separately. I will create issue for that

Reply:
ok I'll include with next issue